.buildscripts/deploy_snapshot.sh

```diff
@@ -17,9 +18,9 @@ elif [ "$CIRCLE_PROJECT_USERNAME" != "$USERNAME" ]; then
 elif [ "$CIRCLE_JDK_VERSION" != "$JDK" ]; then
   # $CIRCLE_JDK_VERSION must be manually set in circle.yml
   echo "Skipping snapshot deployment: wrong JDK. Expected '$JDK' but was '$CIRCLE_JDK_VERSION'."
-else
+elif [ "$CIRCLE_BRANCH" == "main" ] || [ "$CIRCLE_BRANCH" == "develop" ] && [[ "$VERSION" == *SNAPSHOT ]]; then
   echo "Deploying snapshot..."
-  ./gradlew androidSourcesJar androidJavadocsJar uploadArchives -P signing.keyId=$GPG_SIGNING_KEY_ID -Psigning.password=$GPG_SIGNING_KEY_PW -Psigning.secretKeyRingFile=./secring.gpg \
-                           -PSONATYPE_NEXUS_USERNAME=$SONATYPE_NEXUS_USERNAME -PSONATYPE_NEXUS_PASSWORD=$SONATYPE_NEXUS_PASSWORD -PRELEASE_SIGNING_ENABLED=true
+ ./gradlew androidSourcesJar androidJavadocsJar uploadArchives -P signing.keyId=$GPG_SIGNING_KEY_ID -Psigning.password=$GPG_SIGNING_KEY_PW -Psigning.secretKeyRingFile=./secring.gpg \
```

Comment:
we might as well add line breaks to these params while we're modifying this file, i.e.: 
```bash
  ./gradlew androidSourcesJar androidJavadocsJar uploadArchives \
  			-P signing.keyId=$GPG_SIGNING_KEY_ID \
  			-Psigning.password=$GPG_SIGNING_KEY_PW \
  			-Psigning.secretKeyRingFile=./secring.gpg \
        -PSONATYPE_NEXUS_USERNAME=$SONATYPE_NEXUS_USERNAME \
        -PSONATYPE_NEXUS_PASSWORD=$SONATYPE_NEXUS_PASSWORD \
        -PRELEASE_SIGNING_ENABLED=true
```